### PR TITLE
Data from user Profile Toast and rectification in code

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
code been rectified all files changes can be reviewed except the packages installed
Thank you

### CODE REVIEW

File: src/IconsAndAssets/Icons/Github.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+function Github() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="16"
+      height="16"
+      fill="currentColor"
+      className="bi bi-github"
+      viewBox="0 0 16 16"
```
Comment (devsurajrai) [inline src/IconsAndAssets/Icons/Github.jsx:11]: You have done an impressive work but try not to use inline styling.

File: src/IconsAndAssets/Icons/Github.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function Github() {
```
Comment (devsurajrai) [inline src/IconsAndAssets/Icons/Github.jsx:3]: would recommend you to use es6+ syntax.

File: src/Utilities.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  });
+const notifySuccess = (text) =>
+  toast.success(text, {
+    position: "top-left",
+    autoClose: 5000,
+    hideProgressBar: false,
+    closeOnClick: true,
+    pauseOnHover: true,
+    draggable: true,
+    progress: undefined,
```
Comment (devsurajrai) [inline src/Utilities.jsx]: this piece of code is repeating in every toast state, you can jut store this in a constant and use that.

**Review by devsurajrai — COMMENTED** [review]
Everything looks good. No breaking changes are there. Just try to apply the suggestions I have provide in the review. Good Job.

File: src/IconsAndAssets/Icons/Github.jsx (hunk 2 above)
Comment (Dev-AkshatGupta) [reply src/IconsAndAssets/Icons/Github.jsx:3]: Ok will rectify it thanks for reviewing

File: src/IconsAndAssets/Icons/Github.jsx (hunk 1 above)
Comment (Dev-AkshatGupta) [reply src/IconsAndAssets/Icons/Github.jsx:11]: It was just Svg so used them nothing much.

**Review by b0llu — APPROVED** [review]
Just refactoring of code is done so its looking great. Toast is also looking good and you are using separate toast file thats good too. Nothing to suggest you can merge this PR

**Review by aayushjain-code — APPROVED** [review]
Nice Work Akshat you rectified everything and I cannot find any styling suggestions in your code,
few places you can use ES6+ syntax rest everything looks great.
Go ahead and merge it!

**Pull request merged**